functionaltests/src/test/resources/specs/response/HealthResponse.html

```diff
@@ -15,8 +15,6 @@ <h1>Response from <code>/health</code> URL</h1>
             <th c:assertEquals="#result.statusCode">Status Code</th>
             <th c:assertEquals="#result.mimeType">Mime Type</th>
             <th c:assertEquals="#result.bodyStatus">Body status({&quot;status&quot;:200})</th>
-            <th c:echo="#result.bodyTenants">Tenants (info)</th>
-            <th c:assertEquals="#result.bodyTenantsSize">Tenants list size</th>
```

Comment:
@ahalberkamp Why it has been removed? How is this related to the web test fix?

Reply:
Because the new health-check response doesn`t contain any information about tenants anymore... see:https://github.com/commercetools/commercetools-payone-integration/commit/4c74c0f5924920385bbb798243640f8dcb330980#diff-a684bbe1ce55a8f4dea792d69dd8397aL185

And because of the missing tenant-information the functionaltest from above failed.